### PR TITLE
feat(discover): Update intro component with new examples

### PR DESCRIPTION
**Pull request by lynnagara**
Clicking a query also runs the selected query

### CODE REVIEW

**Comment by lynnagara** [comment]
screen shot 2018-12-04 at 12 16 32 pm

File: tests/js/spec/views/organizationDiscover/discover.spec.jsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ describe('Discover', function() {
       expect(queryBuilder.getExternal().aggregations).toEqual([]);
     });
 
-    it('also runs chart query if there are aggregations', async function() {
+    it('also i chart query if there are aggregations', async function() {
```
Comment (billyvg) [inline tests/js/spec/views/organizationDiscover/discover.spec.jsx]:
```suggestion
    it('also runs chart query if there are aggregations', async function() {
```

File: tests/js/spec/views/organizationDiscover/discover.spec.jsx [hunk 2]
```diff
@@ -452,23 +452,31 @@ describe('Discover', function() {
...
+        .find('Button')
         .simulate('click');
-
       const query = queryBuilder.getInternal();
-      expect(query.fields).toEqual(['id']);
-      expect(query.limit).toEqual(10);
+      expect(query.fields).toEqual(['stack.filename']);
+      expect(query.aggregations).toEqual([['count()', null, 'count']]);
+      expect(query.conditions).toEqual([]);
+      expect(queryBuilder.fetch).toHaveBeenCalledTimes(2);
```
Comment (billyvg) [inline tests/js/spec/views/organizationDiscover/discover.spec.jsx]: Why does fetch get called twice?

**Review by billyvg — APPROVED** [review]

File: tests/js/spec/views/organizationDiscover/discover.spec.jsx (hunk 1 above)
Comment (lynnagara) [reply tests/js/spec/views/organizationDiscover/discover.spec.jsx]: argh

File: tests/js/spec/views/organizationDiscover/discover.spec.jsx (hunk 2 above)
Comment (lynnagara) [reply tests/js/spec/views/organizationDiscover/discover.spec.jsx]: queries with aggregations always make a second query populate the charts

**Pull request merged**